testing/load-aware/config.yaml

```diff
@@ -96,6 +102,10 @@ load_aware:
   target_utilization: null
   safe_variance_margin: null
   safe_variance_sensitivity: null
+  scale_test:
+    distribution: null
+    duration: null
+    instances: null
```

Comment:
will the test work with `null` values?
if not (my guess, but I didn't check it all), please put default values that will run a small test (=make `small_poisson_load` the default :))

this way I can run the test without any preset to make sure that I didn't break anything

Reply:
I believe it should since I provided default parameters to the scale test function in `toolbox/load_aware.py` :
```
def scale_test(self, distribution="poisson", duration=60.0, instances=10):
```

I can add defaults to the config.yaml though if I'm wrong or if that'd be preferred :)

Reply:
see my other comment,
* currently these values _are_ used, because you're calling the command without any parameter
* but the config file settings aren't used either
* when you'll use `from_config` to pass the values from the config file ... your command will crash, because it will receive `None` values for the settings above